applications/StructuralMechanicsApplication/custom_elements/membrane_element.cpp

```diff
@@ -0,0 +1,1271 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Klaus B. Sautter
+//
+
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "includes/checks.h"
+#include "utilities/math_utils.h"
+#include "custom_elements/membrane_element.hpp"
+#include "structural_mechanics_application_variables.h"
+#include "custom_utilities/structural_mechanics_element_utilities.h"
+
+namespace Kratos
+{
+
+// Constructor
+MembraneElement::MembraneElement( IndexType NewId, GeometryType::Pointer pGeometry )
+    : Element( NewId, pGeometry )
+{
+
+}
+
+// Constructor
+MembraneElement::MembraneElement(IndexType NewId, GeometryType::Pointer pGeometry, PropertiesType::Pointer pProperties)
+    : Element( NewId, pGeometry, pProperties )
+{
+
+}
```

Comment:
In the future I suggest to leave empty constructors in the header, this makes it more readable